lexos/processors/analyze/dendrogrammer.py

```diff
@@ -247,13 +240,9 @@ def get_augmented_dendrogram(*args, **kwargs):
             pyplot.annotate("%0.4g" % y, (x, y), xytext=(0, -8),
                             textcoords='offset points',
                             va='top', ha='center', size='small')
-    pyplot.legend(
-        p,
-        ['the branch height legend'],
-        numpoints=1,
-        bbox_to_anchor=(
-            1.1,
-            1.1))
+
+    pyplot.legend(p, ['the branch height legend'], numpoints=1,
+                  bbox_to_anchor=(1.1, 1.1))
 
 
 def trim(im: Image.Image) -> Image.Image:
```

Comment:
This function does not consistently return (if `bbox == False`, then there is no return), why?